vimeo-networking/src/main/java/com/vimeo/networking/model/Membership.java

```diff
@@ -0,0 +1,95 @@
+package com.vimeo.networking.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * This object contains user membership information.
+ * <p>
+ * Created by brentwatson on 12/3/18.
```

Comment:
unnecessary created by, also applies to other files.

Reply:
Again, I was trying to follow convention. All Java DTOs have this but Kotlin DTOs do not, so I did the same.